content/posts/android-tv-14.md

```diff
@@ -0,0 +1,110 @@
+
+![Screenshot of the new login design](/images/posts/android-tv-14/login-1.png)
+
+## Support for external trailers
+
+The newly added support for external trailers allows the playback of trailers from YouTube, Vimeo and other video services. Previously only local trailers could be played. This feature is only available when a browser or video service app is installed to open the trailer.
+
+Unfortunately this feature has some limitations. Some popular apps, like YouTube, don't make it easy to go back to the Jellyfin app. And in some television platforms you might see an error telling you there is no app to open the trailer. If this happens you should check if the YouTube/Vimeo/other app is installed. For an optimal experience we recommend using local trailers that play inside the Jellyfin app.
+
+## Changes to the Live TV guide
+
+The Live TV guide got some long awaited fixes and design tweaks. These include, but are not limited to:
+
+- Fixed the incorrect sizing of channel images in some places
+- Fixed an issue where the guide could have some sizing problems with many channels
+- Fixed the "null" text when information was missing in a some places
+- Fixed guide sorting not always working
+- Removed the category images causing contrast issues
+- Restyled some buttons to use theme colors
+- Restyled the filter and options popups to the same design as other parts of the app
+
+## Improved browsing grid
+
+To make it easier to see a lot of items or just a few we have added two new image sizes in the grid view: smallest and extra large. Together with this change the sizing is now based on the actual size of your device, fixing some bugs on 4k displays. This might change the size you're used to a bit. We also fixed some issues with the vertical direction and now consider it a stable feature!
+
+## New crash reporting
+
+Back in 2020 with version 0.11 we added a crash reporting utility. This function helped us fix a lot of issues with the app. The crash reporting relied on a third-party service to collect the reports. We kept tight control of who could access this data, but never liked this approach as it did use a third-party service we have no control over. In this update the crash reporting code was rewritten and now sends all reports to your own Jellyfin server. The crash reports use a new format that makes it more readable for both the user and developers and includes details to identify and fix bugs even faster.
+
+Crash reporting is now turned on by default. When the app crashes it will send a report to the latest server you used. On fresh installations the report is queued until a server is connected to.
```

Comment:
```suggestion
Crash reporting is now turned on by default. When the app crashes it will send a report to the last server you used. On fresh installations the report is queued until connecting to a server.
```